CHANGELOG.md

```diff
@@ -0,0 +1,23 @@
+# Orchid Changelog
+
+All notable changes to Orchid will be documented in this file.
+
+The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
+and this project adheres to [Semantic
+Versioning](https://semver.org/spec/v2.0.0.html).
+
+## [Unreleased]
```

Comment:
Could you copy in the template like I have in ONI? Want to make sure the diff link is in there too. I guess compare the next release against 3.0.1?

Reply:
I can copy in the template, yes.  I was waiting to add the diff link until this was actually going to be released, since I hadn't done the normal steps of bumping the version to prep for a release and such, but I hadn't forgotten it!